docs/content/topics/project-developers/events.md

```diff
@@ -9,4 +9,229 @@ aliases:
+that are labeled in a particular way. For example, a label of `branch: main`
+might be used on an event subscription to limit inbound GitHub events to only
+those that occur on the `main` branch of a given repo.
+
+### Short Title
+
+A short title may be provided for an event. This can then be utilized for
+logging, categorization or visual representation purposes, etc.
+
+### Long Title
+
+A longer, more descriptive title may be provided for an event. This may be
+helpful for providing additional context for users consuming event details.
+
+### Source State
+
+Source State for an event is a key/value map representing event state that can
+be persisted by the Brigade API server so that gateways can track event
+handling progress and perform other actions, such as updating upstream
+services. For example, the GitHub Gateway may utilize this field for tracking
+completion of a CI run on a pull request, in order to update GitHub with logs
+and pass/fail status.
+
+### Summary
+
+Whereas an event payload represents arbitray information sent inbound to the
+client handling the event, the event summary is meant to relay outbound
+context/details generated during handling of the original event. The gateway
+responsible for emitting the event then has access to a summary of the work
+done while processing this event.
+
+To explore the SDK definitions of an Event object, see the [Go SDK Event] and
+[JavaScript/TypeScript SDK Event].
+
+[GitHub Gateway]: https://github.com/brigadecore/brigade-github-gateway
+[Go SDK Event]: https://github.com/brigadecore/brigade/blob/v2/sdk/v2/core/events.go
+[JavaScript/TypeScript SDK Event]: https://github.com/brigadecore/brigade-sdk-for-js/blob/master/src/core/events.ts
+
+## Event Subscriptions
+
+In order to receive events, a Brigade project must explicitly subscribe to
+them. This is done via an Event Subscription configured on the project.
+
+For an overview of a project definition in Brigade, see the [Projects] doc.
+
+Here we'll look at a sample project and dig into its `eventSubscriptions`
+section:
+
+```yaml
+apiVersion: brigade.sh/v2-beta
+kind: Project
+metadata:
+  id: hello-world
+description: Demonstrates responding to an event with brigadier
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/cli
+    types:
+    - exec
+```
+
+The `hello-world` project above subscribes to one event, that of source
+`brigade.sh/cli` and type `exec`. This is the event emitted from the
+`brig event create` command.
+
+Additional events and/or additional types of a given event are added under this
+`eventSubscriptions` section. [Qualifiers] and [Labels] may also be added, via
+the `qualifiers` and `labels` fields respectively, to further refine inbound
+events.
+
+Here's a look at an event subscription configuration using all of the above:
+
+```yaml
+eventSubscriptions:
+- source: brigade.sh/github
+  qualifiers:
+    repo: brigadecore/brigade
```

Comment:
I vaguely recall that we had a discussion about not using our own repos as examples because it could introduce confusion.

I can't recall if we took any other action after that convo or if it was just something to keep an eye out for going forward.

At any rate, I think seeing the word "brigade" in reference to userland things might improve the clarity of the examples.